controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/routing/responses/endpoint/endpoints.json

```diff
@@ -0,0 +1,31 @@
+{
+  "endpoints": [
+    {
+      "endpoint": "default",
+      "dns": "a1--t1.global.vespa.oath.cloud",
```

Comment:
How is this going to be used? Perhaps better to include the URL? Otherwise console won't know about scheme or port.

Reply:
We aren't really going to use those values much from the console. We only care about the name and the status for the zone for now. I just added the other values for completeness.